lib/brady.ex

```diff
@@ -81,7 +81,7 @@ defmodule Brady do
   defp render_with_options(markup, []), do: {:safe, markup}
   defp render_with_options(markup, options) do
     markup
-    |> Floki.parse
+    |> Floki.parse_fragment!()
```

Comment:
Should we use [`Floki.parse_fragment/1`](https://hexdocs.pm/floki/Floki.html#parse_fragment/1) instead of the `!` version? I think that would be more equivalent to the behavior of [`parse/1`](https://hexdocs.pm/floki/Floki.html#parse/1).

Reply:
@WilHall - Ah! Yeah, totally, I got those two confused. Fixed here: https://github.com/thoughtbot/brady/pull/21/commits/dbcbefedd2318a31c7babda5f72f99fbd3c10912